app/views/faculties/student_list.html.erb

```diff
@@ -0,0 +1,81 @@
+<div class="button hover:underline">
+  <%= link_to 'Back to courses list', faculty_path, class: "btn btn-sm btn-blue mr-1" %>
+</div>
+
+<p>
+  <strong>Course Title:</strong>
+  <%= @course.display_name %>
+</p>
+
+<p>
+  <strong>Available spaces:</strong>
+  <%= @course.available_spaces %>
+</p>
+
+<p>
+  <strong>Status:</strong>
+  <%= @course.status %>
+</p>
+
+<p>
+  <strong>Student List:</strong>
+</p>
+
+<table class="px-2 min-w-full w-full divide-y divide-gray-300">
+  <thead class="bg-gray-300">
+    <tr>
+      <th scope="col" class="py-3.5 text-sm font-semibold text-gray-900 text-center">Name</th>
+      <th scope="col" class="pl-4 py-3.5 text-center text-sm font-semibold text-gray-900 lg:table-cell">Email</th>
+      <th scope="col" class="pl-4 py-3.5 text-center text-sm font-semibold text-gray-900 lg:table-cell">Gender</th>
+      <th scope="col" class="pl-4 py-3.5 text-center text-sm font-semibold text-gray-900 lg:table-cell">Year in School</th>
+      <th scope="col" class="pl-4 py-3.5 text-center text-sm font-semibold text-gray-900 lg:table-cell">Country</th>
+      <th scope="col" class="pl-4 py-3.5 text-center text-sm font-semibold text-gray-900 lg:table-cell">City - State</th>
+      <th scope="col" class="pl-4 py-3.5 text-center text-sm font-semibold text-gray-900 lg:table-cell"></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @students.each do |student| %>
+      <% if @students.index(student).odd? %>
+        <tr class="relative py-2 pl-4 sm:pl-6 pr-3 text-sm border-t border-grey-700 bg-gray-200">
+      <% else %>
+        <tr class="relative py-2 pl-4 sm:pl-6 pr-3 text-sm border-t border-grey-700 bg-white">
+      <% end %>
+        <td class="relative py-2 pl-4 sm:pl-6 pr-3 text-sm">
+          <div class="font-medium text-gray-900">
+            <%= student.applicant_detail.full_name %>
+          </div>
+        </td>
+        <td class="relative py-2 pl-4 sm:pl-6 pr-3 text-sm">
+          <div class="font-medium text-gray-900 hover:underline">
```

Comment:
Since you have the "more info" now, maybe drop the hover:underline here

Reply:
Oops, I was sure I did that. Thanks for noticing.